ember.js/2018-03/Meeting-Ember-Core-Team-20180316.md

```diff
@@ -1,29 +0,0 @@
-# Meeting: Ember Core Team 2018/03/16
-
-## Agenda
-  Previous week: +Meeting: Ember Core Team 2018/03/09 
-
-
-- (public) 
-- (private)
```

Comment:
was this a holiday? we might want to say that

Reply:
The next day was st patricks, but not on the 16th AFAICT